docs/DeveloperGuide.adoc

```diff
@@ -98,9 +93,9 @@ image::LogicClassDiagram.png[]
 *API* :
```

Comment:
The text in the diagrams seems too small, perhaps the arrows could be shorter so the diagram can be more zoomed in?
![image](https://user-images.githubusercontent.com/26473264/67543141-898a4600-f722-11e9-99fe-5707d1fc41e0.png)